scripts/update_bootlint.js

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+
+/* eslint-env shelljs */
+
+'use strict';
+
+require('shelljs/global');
+var https       = require('https');
+var fs          = require('fs');
+var path        = require('path');
+
+var version     = process.argv[2];
+
+var basedir     = path.join(__dirname, '..');
+var bootlintDir = path.join(basedir, 'public', 'bootlint', version);
+var UGLIFYJS    = path.join(basedir, 'node_modules/.bin/uglifyjs');
+
+if (!version) {
+    echo('Valid Bootlint version required.');
+    process.exit(1);
+}
+
+// strip leading 'v' if present
+version = version.replace(/^v/, '$1');
```

Comment:
Is `$1` correct here? Shouldn't it be `version.replace(/^v/, '')`? Perhaps this is a shell js thing I'm not aware of?